friends/src/Components/Friend.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+function Friend() {
+
+
+    return (
+        <h2>hello</h2>
+    )
```

Comment:
This is very small for a component - is this a test file that you forgot to take out?